### DISCUSSION TITLE
Featured Image issues

### DISCUSSION THREAD

**Discussion by tresero** [razonyang/hugo-theme-bootstrap#836] [General]
If you have the images front matter tag, and are using page bundles, the featured image doesn't show. I may be doing something wrong, but it works perfectly witn no images front matter.

Take a page, /articles/rare-percussion-sample
I have featured.webp in the folder and this in the front matter.

images: 
  - featured.webp
  - Okonkolo.webp
  - Quijada.webp
  - saltanes.webp
  - Dominican-Clave.webp
  - Shell.webp

audio: []

**Comment by razonyang** [comment]
Hi, that's a limitation of some Hugo built in templates, that any images specified in front matter (highest priority) are treated as static images or external images, that is, the page images resource won't be used if `images` presents in front matter.

So we can either remove the `images` param, the theme will recognize page resources those names begin with `feature`. Or leave `images` parameter as it is and move your images to the static folder.

The downside of static images is that, the theme is unable to resize them.

**Comment by tresero** [comment]
Okay, no problem. I was only using it for the OG image stuff. I'll figure a way around it.
